Gruvo/Gruvo/appsettings.json

```diff
@@ -6,6 +6,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "GruvoMSSQL": "Data Source=.\\;Initial Catalog=Gruvo;Integrated Security=True"
```

Comment:
Stop modifying this) I have local changes with my DB and can pull new changes while you don't change it. Once you do - I have conflicts (and everyone has)